### PR TITLE
Create de.json

### PR DESCRIPTION
**Pull request by davidvanbeveren**

### CODE REVIEW

File: lang/de.json [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  "resources.books.kind.hardcover": "Gebundene Ausgabe",
+  "resources.books.kind.digital": "Digital",
+  "resources.books.ethics.title": "Wildtierethik",
+  "resources.books.ethics.description": "Obwohl viele Ethiker die Intuition haben, dass wir die Natur in Ruhe lassen sollten, argumentiert Kyle Johannsen, dass wir die Pflicht haben, sichere Möglichkeiten zur groß angelegten Hilfe für Wildtiere zu erforschen. Mit Konzepten aus der moralischen und politischen Philosophie analysiert Johannsen das Problem des Leidens von Wildtieren und untersucht, wie eine kollektive, institutionelle Verpflichtung zur Hilfe für Wildtiere verstanden werden sollte.",
+  "resources.books.making_a_stand.title": "Stellung beziehen für Tiere",
+  "resources.books.making_a_stand.description": "Dieses Buch ist anregend und provokativ und untersucht, wie Menschen andere Tiere sehen und behandeln und argumentiert dafür, dass wir allen Wesen, menschlichen und nichtmenschlichen, gleiche Berücksichtigung und Respekt entgegenbringen sollten. Unsere Welt ist von "Ismen" wie Rassismus und Sexismus geplagt, aber wir haben möglicherweise einen sehr wichtigen übersehen: Speziesismus.",
```
Comment (stephan281094) [inline lang/de.json]: This is invalid JSON now. Double quotes need to be escaped: `Unsere Welt its von \"Ismen\" wie Rassismus`

File: lang/de.json [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  "resources.podcasts.knowing_animals.description": "In dieser Episode haben wir Dr. Kyle Johannsen zu Gast. Kyle ist außerordentlicher Professor für Philosophie an der Queen's University in Kanada. Wir diskutieren sein Buch Wild Animal Ethics, das 2020 von Routledge veröffentlicht wurde.",
+  "resources.podcasts.wildness.title": "Wer kümmert sich um Wildtiere? mit Maia Laperle",
+  "resources.podcasts.wildness.description": "Maia stellt die Werte und Grundsätze des Wohlergehens von Wildtieren vor und interviewt Führungskräfte im Bereich des Wohlergehens von Wildtieren. In jeder Folge erkundet Moderatorin Maia Laperle grundlegende Konzepte des Wohlergehens von Wildtieren und was wir über das Leben in der Wildnis wissen und nicht wissen.",
+  "resources.podcasts.the_reality_check.title": "Brian Tomasik über das Leiden von Wildtieren",
+  "resources.podcasts.the_reality_check.description": "Unser Gast Brian Tomasik öffnet uns die Augen für das Leiden von Wildtieren. Brian Tomasik schreibt über Ethik, Tierschutz und Zukunftszenarien aus einer leidensorientierten Perspektive auf seiner Hauptwebsite "Essays On Reducing Suffering".",
```
Comment (stephan281094) [inline lang/de.json]: Same here: `auf seiner Hauptwebsite \"Essays On Reducing Suffering\".`

**Review by stephan281094 — CHANGES_REQUESTED** [review]

**Review by stephan281094 — APPROVED** [review]
Looks good to me.

Do keep in mind that additional work is required before the German version becomes available on the website:
- Create `pages/de.tsx`.
- Update `i18n.ts` to include the new language.
- Update `components/header.tsx` to include the new language in the language switcher.

These are things I can pick up later today if that's more convenient 🙃

**Comment by davidvanbeveren** [comment]
Thanks! Wanted to do myself just to know the process better, thanks for offering!

File: components/header.tsx [hunk 3]
```diff
@@ -24,6 +24,7 @@ export function Header({ locale = "en" }: Props) {
       <Select defaultValue={locale} onChange={onLangChange}>
         <option value="en">En</option>
         <option value="fr">Fr</option>
+        <option value="de">De</option>
```
Comment (stephan281094) [inline components/header.tsx]: Nit: Having these sorted alphabetically makes it easier for users to find their language:
```suggestion
        <option value="de">De</option>
        <option value="en">En</option>
        <option value="fr">Fr</option>
```

File: i18n.ts [hunk 4]
```diff
@@ -3,11 +3,13 @@ import { initReactI18next } from "react-i18next";
 
 import en from "./lang/en.json";
 import fr from "./lang/fr.json";
+import de from "./lang/de.json";
 
 export const defaultNS = "translation";
 export const resources = {
   en: { [defaultNS]: en },
   fr: { [defaultNS]: fr },
+  de: { [defaultNS]: de },
```
Comment (stephan281094) [inline i18n.ts]: Same here:
```suggestion
import de from "./lang/de.json";
import en from "./lang/en.json";
import fr from "./lang/fr.json";

export const defaultNS = "translation";
export const resources = {
  de: { [defaultNS]: de },
  en: { [defaultNS]: en },
  fr: { [defaultNS]: fr },
```

**Review by stephan281094 — APPROVED** [review]
Some nitpicks, but other than that I think we're good to go 👍

**Pull request merged**